Gemfile

```diff
@@ -60,3 +60,4 @@ gem "breadcrumbs_on_rails"
 gem 'ransack', github: 'activerecord-hackery/ransack'
 gem 'stripe'
 gem 'stripe_event'
+gem 'sidekiq'
```

Comment:
can you add versions to gems? like the current version because new versions may break some staff